client/lib/wpcom-undocumented/lib/me.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Module dependencies.
  */
-import Me from 'wpcom/dist/lib/me';
+import Me from 'wpcom/build/lib/me';
```

Comment:
> It's lame to rely on an implementation detail like this.

In this case should we sport the all libraries from the core?
I wonder why we are using this path instead of `wpcom/lib/me`. And in other hand `lodash` uses this way to import libraries.